server/database/createSeed.js

```diff
@@ -49,23 +54,44 @@ const generateAuthors = function(n) {
 };
 
 //consecutive IDs, static photo
-const generatePhotos = function(n) {
+const generatePhotos = function(n, start = 0) {
   let photos = [];
-  for (let i = 0; i < n; i++) {
+  for (let i = start; i < n; i++) {
     let photo = {
       id: i,
       url: faker.image.image()
```

Comment:
Ensure you create an S3 bucket containing 1000 images for your final seed. They want us to get experience using the S3 buckets and creating our own data. Faker images are usually using randomizing APIs, thus it will not be a static image.

Reply:
tentatively based upon my own machine it shouldn't take up more than half a gig or so. It seems to know to garbage collect and I'm on the articles portion of saving now and node is still at less than half a gig of ram... and most of the variables are local so I think if the garbage collection is good it should never go above the ram usage during the articles loading as that's when potentially the largest things are created that might be lying around for a while.